.editorconfig

```diff
@@ -10,8 +10,9 @@ indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-[{*.yml,package.json}]
+[{*.yml,*.ss,*.css,*.json}]
```

Comment:
Just make it 4 spaces everywhere... ;-)

Reply:
Done and done :)

Reply:
I expect some pushback from the core team as it's not a part of the functional change, but tbh I never understood the need for 2 spaces for `.yml` and `.json` only.

Reply:
Just copy whatever is in https://github.com/silverstripe/silverstripe-framework/blob/4/.editorconfig

> the need for 2 spaces

I don't know either, though consistent standards mean we don't need to think or spend time debating stuff =)

Reply:
@emteknetnz  Editor config updated from framework